### PR TITLE
Feature 2

### PR DESCRIPTION
**Pull request by Namrata-J**
#2 here i have refacored the existing code, and have worked on the responsiveness the the screens.

please review it, thankyou!!

### CODE REVIEW

File: src/components/FilterAndSortPopUp/filterAndSortPopUp.css [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    }
+
+    .tn_filter-popUp-container {
+        max-width: 70%;
+    }
+}
+
+@media only screen and (max-width: 450px) {
+    
+}
```
Comment (tusharkandpal) [inline src/components/FilterAndSortPopUp/filterAndSortPopUp.css]: please remove this media query

**Review by tusharkandpal — APPROVED** [review]
clear & well organized structure.. !

Suggestion: you can name css file same as jsx.

File: src/components/FilterAndSortPopUp/filterAndSortPopUp.css (hunk 1 above)
Comment (Namrata-J) [reply src/components/FilterAndSortPopUp/filterAndSortPopUp.css]: thanks for pointing out!! will remove it!!

File: src/contexts/header-context.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        popUpSearchBarDisplay === "none" ? setPopUpSearchBarDisplay("flex") : setPopUpSearchBarDisplay("none")
+    }
+
+    const getThePopUpFilterAndSortDisplay =() => {
+        popUpFilterAndSortDisplay === "none" ? setPopUpFilterAndSortDisplay("flex") : setPopUpFilterAndSortDisplay("none")
+    }
+
+    const closeThePopUpFilterAndSort = () => {
+        setPopUpFilterAndSortDisplay("none")
+    }
```
Comment (mariyasada) [inline src/contexts/header-context.js:20]: good naming convention

**Review by mariyasada — APPROVED** [review]
The code structure is good

**Pull request merged**